### DISCUSSION TITLE
java22 pom variant

### DISCUSSION THREAD

**Discussion by avlo** [tcheeric/nostr-java#175] [General]
@tcheeric 

hi, eric.  per your interest/usefulness, a [nostr-java java22 pom variant](https://github.com/avlo/nostr-java-avlo-fork/commit/a5ab118689975e689e16b167da49f97149f6efb) exists.  changes were limited to 7 lines in the parent pom.

the variant builds and tests without issue here, as well as 100% functioning with SuperConductor upgrade to virtual threads (java 21+)).

**Comment by dukeh3** [comment]
Yes I get this, but would it not be better to cap it at java 21, instead of 22 as per what I have seen everything works well with 21.
